pkg/grub/patches-aarch64-2.06/0007-set-cmddevice.patch

```diff
@@ -0,0 +1,26 @@
+From 35c2f5ba467711b308124ebb61ea6aef4eb77707 Mon Sep 17 00:00:00 2001
+From: Petr Fedchenkov <giggsoff@gmail.com>
```

Comment:
Did you copy this patches from somewhere or you actually implemented them?

Reply:
They are from [here](https://github.com/lf-edge/eve/blob/master/pkg/grub/patches/0000-core-os-merge.patch).